src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +611,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(@StringRes int name, String flag) {
+		return flag + " " + getString(name);
```

Comment:
I'm not sure I like the messed up order this causes although I agree that flags look better in the front.